rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary can use Windows Management Instrumentation (WMI) to install event filters, providers, consumers, and
+bindings that execute code when a defined event occurs. Adversaries may use the capabilities of WMI to subscribe to an
+event and execute arbitrary code when that event occurs, providing persistence on a system.
+"""
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Windows Management Instrumentation Event Subscription"
+risk_score = 21
+rule_id = "9b6813a1-daf1-457e-b0e6-0bb4e55b8a4c"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  (process.name : "wmic.exe" or process.pe.original_file_name == "wmic.exe") and
+  process.args : "create"
```

Comment:
create method can be used also to create a process and unrelated to wmi persistence creation, below example of wmic (needs to add more args for context e.g. ...  and process.args: ("CommandLineEventConsumer", "ActiveScriptEventConsumer") below example of cmdline:

C:\Windows\System32\wbem\WMIC.exe" /namespace:"\\root\subscription" PATH CommandLineEventConsumer CREATE Name="BotConsumer23", ExecutablePath="c:\Windows\System32\cmd.exe", CommandLineTemplate="c:\Windows\System32\cmd.exe"